cmd/loadtest/loadtest.go

```diff
@@ -380,6 +380,12 @@ func initializeLoadTestParams(ctx context.Context, c *ethclient.Client) error {
 		return err
 	}
 
+	chainID, err := c.ChainID(ctx)
+	if err != nil {
+		log.Error().Err(err).Msg("Unable to fetch chain ID")
+		return err
+	}
+
```

Comment:
If the user provides a chainID, this call is not used. It might be nice to log it out for visibility. Otherwise, I would only get the chain id if the user doesn't provide it.

```suggestion
	chainID, err := c.ChainID(ctx)
	if err != nil {
		log.Error().Err(err).Msg("Unable to fetch chain ID")
		return err
	}
	log.Trace().Uint64("chainID", chainID.Uint64()).Msg("Detected Chain ID")

```